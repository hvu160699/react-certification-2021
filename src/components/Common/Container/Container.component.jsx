import styled from '@emotion/styled';

const Container = styled.div`
  width: 100%;

  @media (min-width: 640px) {
    max-width: 540px;
  }

  @media (min-width: 768px) {
    max-width: 720px;
  }

  @media (min-width: 1024px) {
    max-width: 960px;
  }

  @media (min-width: 1280px) {
    max-width: 1140px;
  }
`;

export default Container;
