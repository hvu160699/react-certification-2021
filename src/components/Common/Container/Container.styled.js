import styled from '@emotion/styled';

const Container = styled.div`
  width: 100%;

  @media (min-width: 1280px) {
    max-width: 1280px;
  }
`;

Container.displayName = 'Container';

const Styled = {
  Container,
};

export default Styled;
