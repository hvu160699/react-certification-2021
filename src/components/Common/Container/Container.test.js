import React from 'react';

import renderer from 'react-test-renderer';

import Container from './Container.component';

describe('Container', () => {
  it('Shoul render a container that can contain multiply element', () => {
    const container = renderer
      .create(
        <Container>
          <div>Find the button</div>
          <button type="button">Click me</button>
        </Container>
      )
      .toJSON();

    expect(container).toMatchSnapshot();
  });
});
