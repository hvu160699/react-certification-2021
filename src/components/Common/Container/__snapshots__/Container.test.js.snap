// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Shoul render a container that can contain multiply element 1`] = `
<div
  className="css-1avkbq0"
>
  <div>
    Find the button
  </div>
  <button>
    Click me
  </button>
</div>
`;
